### PR TITLE
Only enable wifi tests where available on openSUSE

### PR DESCRIPTION
**Pull request by okurz**
Related progress issue: https://progress.opensuse.org/issues/29369

### CODE REVIEW

**Comment by okurz** [comment]
check failure fixed

File: lib/main_common.pm [hunk 1]
```diff
@@ -546,7 +546,9 @@ sub load_extra_tests {
         # well
         if (check_var('DESKTOP', 'gnome')) {
             loadtest 'x11/yast2_lan_restart';
-            if (!is_sle || sle_version_at_least('12-SP4')) {
+            # we only have the test dependencies, e.g. hostapd available in
+            # openSUSE
+            if (check_var('DISTRI', 'opensuse')) {
```
Comment (asmorodskyi) [inline lib/main_common.pm]: we have function called is_leap , what reason not to use it here ?

**Comment by okurz** [comment]
There is no `is_opensuse`. This is what we would need for tw & leap

**Comment by asmorodskyi** [comment]
With this change you duplicating combination of conditions defined in lines 520-521 what if you just move this tests there ?

**Comment by okurz** [comment]
In theory we could move the section into the same "is_opensuse" section but then it would turn around the logic, network related tests would not be called in the same place anymore and also the comment above the code would not fit anymore. This is going too far. @nicksinger said "ok"

**Pull request merged**